src/libs/API.js

```diff
@@ -132,13 +132,31 @@ function handleExpiredAuthToken(originalCommand, originalParameters, originalTyp
         })
         .catch(() => (
 
-            // If the request did not succeed and we were not logged out requeue the original request. We are using
-            // request() here so that it we can reauthenticate again when it fails
-            // eslint-disable-next-line no-use-before-define
-            request(originalCommand, originalParameters, originalType)
+            // If the request did not succeed because of a networking issue or the server did not respond requeue the
+            // original request.
+            Network.post(originalCommand, originalParameters, originalType)
         ));
 }
 
+// Register a method with Network to call when the authToken has expired.
```

Comment:
This comment isn't really necessary since it doesn't add any helpful information